plugins/genesys-gl32xx/genesys-gl32xx.quirk

```diff
@@ -0,0 +1,7 @@
+[BLOCK\VEN_05E3&DEV_0749]
+Plugin = genesys_gl32xx
+FirmwareSize = 0x10000
+
+[BLOCK\VEN_05E3&DEV_0764]
+Plugin = genesys_gl32xx
+FirmwareSize = 0x1C000
```

Comment:
does the size depend on the *device type* or the *SPI chip*?

Reply:
No answer here.
Due implicit signs I think it is for the device, but I'll better check.

Reply:
Seems device specific; sorry for the noise.